src/page-modules/assistant/layout.tsx

```diff
@@ -213,11 +217,20 @@ function AssistantLayout({
             mode="interactive_0"
             disabled={!selectedFromFeature || !selectedToFeature}
             buttonProps={{ type: 'submit' }}
+            state={isSearching ? 'loading' : undefined}
           />
         </div>
       </form>
 
-      <section className={style.contentContainer}>{children}</section>
+      <section className={style.contentContainer}>
```

Comment:
Maybe we don't need to swap out the children if there are search results present already?

I like the loading state in the button, and the "Loading search results..." could also be nice, but I don't necessarily think that we need to show it when there are results there.

Reply:
I agree! I've updated to reflect this now 👍